src/screens/wallet-settings/wallet-settings.screen.tsx

```diff
@@ -39,20 +58,26 @@ export const WalletSettingsScreen = ({ history }: any) => {
           <StyledInput type='text' label='Wallet Name' placeholder='A test Wallet' />
 
           <BeneficiaryContainer>
-            <Label>Add a beneficiary</Label>
-            <AddIconContainer align={'end'}>
-              <IconSvg name='add' size='large' />
-            </AddIconContainer>
+            <Label> Wallet Beneficiary</Label>
           </BeneficiaryContainer>
           <Box marginTop={-2} marginBottom={1.2}>
             <StyledInput type='text' placeholder='johndoe@safeint.com' />
           </Box>
 
-          <Label>Advanced options</Label>
-          <Box row hCenter marginTop={-2} justify={'between'}>
-            <Label>Signaling Period</Label>
-            <SignnalingInput type='text' placeholder='10 days' />
-          </Box>
+          <Accordion label='Advanced Options'>
+            <Box row hCenter marginTop={2} justify={'between'}>
+              <Label>Signaling Period</Label>
+              <SignnalingInput type='text' placeholder='10 days' />
+            </Box>
+            <Box marginTop={2}>
+              <DropDown
+                placeholder='select network'
+                label='Select Network'
+                options={networkOptions}
+                onChange={(e: any) => setSelectNetwork(e)}
```

Comment:
The font of the option once selected is small and different from the default. Can you check this?

Reply:
oh yeah. working on it.